test/powershell/Modules/Microsoft.PowerShell.Management/New-PSDrive.Tests.ps1

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Tests for New-PSDrive cmdlet." -Tag "CI","RequireAdminOnWindows" {
+    Context "Validate New-PSDrive Cmdlet with -Persist switch." {
+        BeforeEach {
+            $PSDriveName = "W"
+            $RemoteShare = "\\$env:COMPUTERNAME\$($env:SystemDrive.replace(':','$\'))"
+        }
+
+        AfterEach {
+            Remove-PSDrive -Name $PSDriveName -Force -ErrorAction SilentlyContinue
+        }
+
+        It "Should not throw exception for persistent PSDrive creation." -Skip:(-not $IsWindows) {
+            { New-PSDrive -Name $PSDriveName -PSProvider FileSystem -Root $RemoteShare -Persist -ErrorAction Stop } | Should -Not -Throw
+        }
+
+        it "Should throw exception if root is not a remote share." -Skip:(-not $IsWindows) {
+            { New-PSDrive -Root "TestDrive:\" -PSProvider FileSystem -Name $PSDriveName -Persist -ErrorAction Stop } | Should -Throw -ErrorId 'DriveRootNotNetworkPath'
```

Comment:
Should keep the parameters in the same order to make it easier to spot differences.  On line 16, you use Name, PSProvider, Root, Persist, Erroraction.  Just keep that order.